target/linux/realtek/image/rtl839x.mk

```diff
@@ -1,5 +1,13 @@
 # SPDX-License-Identifier: GPL-2.0-only
 
+define Device/d-link_dgs-1210-52
+  $(Device/d-link_dgs-1210)
+  SOC := rtl8393
+  DEVICE_MODEL := DGS-1210-52
+  DEVICE_PACKAGES += uboot-envtools
```

Comment:
Have you checked if you need to have uboot-envtools as device package? When I put my -10mp in the realtek config file for uboot-envtools this package became a dependable package and auto included.